example_config.json

```diff
@@ -0,0 +1,40 @@
+{
+    "debug": false,
+    "use_real_mongo": true,
+    "mongo_database": "optimade",
+    "mongo_uri": "mongodb+srv://MongoUser:MongoPassword@FakeMongoServer.com/optimade_database",
```

Comment:
Will this work if we use this config file in CI instead of `tests/test_config.py` - or does it actually make more sense to use the test config in tests ... I guess it does.

Reply:
The idea is to have something that looks a bit more like how someone might use it. Mostly the test file and the example file will look similar, but I felt the slight differences did warrant a new file.